subgraphs/ens-governance/schema.graphql

```diff
@@ -0,0 +1,151 @@
+  totalDelegates: BigInt!
+  "Total number of votes delegated expressed in the smallest unit of the Compound Token"
+  delegatedVotesRaw: BigInt!
+  "Total number of votes delegated expressed as a BigDecimal normalized value for the Compound Token"
+  delegatedVotes: BigDecimal!
+
+  "Number of proposals currently queued for execution"
+  proposalsQueued: BigInt!
+  "Number of proposals currently executed"
+  proposalsExecuted: BigInt!
+  "Number of proposals currently canceled"
+  proposalsCanceled: BigInt!
+
+  "Quorum fraction numerator value. (quorum = totalSupply * numerator / denominator)"
+  quorumNumerator: BigInt
+  "Quorum fraction denominator value. (quorum = totalSupply * numerator / denominator)"
+  quorumDenominator: BigInt
+}
+
+type Proposal @entity {
+  id: ID!
+  "This is the on-chain ID associated with the proposal"
+  proposalId: BigInt!
+  "Governor Alpha Proposal description in markdown format"
+  description: String
+  "Delegate that proposed the change"
+  proposer: Delegate!
+  "State of the proposal"
+  state: ProposalState!
+
+  "Votes against the proposal"
+  againstVotes: BigInt!
+  "Votes for the proposal"
+  forVotes: BigInt!
+  "Votes abstaining to the proposal"
+  abstainVotes: BigInt!
+  "Votes associated to this proposal"
+  votes: [Vote!]! @derivedFrom(field: "proposal")
+
+  "Block number proposal was created in"
+  creationBlock: BigInt!
+  "Timestamp of block proposal was created in"
+  creationTime: BigInt!
+  "Block number from where the voting starts"
+  startBlock: BigInt!
+  "Block number from where the voting ends"
+  endBlock: BigInt!
+  "Once the proposal is queued for execution it will have an ETA of the execution"
+  executionETA: BigInt
+  "Block number proposal was executed in"
+  executionBlock: BigInt
+  "Timestamp of block proposal was executed in"
+  executionTime: BigInt
+  "Block number proposal was canceled in"
+  cancellationBlock: BigInt
+  "Timestamp of block proposal was canceled in"
+  cancellationTime: BigInt
+
+  "Targets data for the change"
+  targets: [String!]
+  "Values data for the change"
+  values: [BigInt!]
+  "Signature data for the change"
+  signatures: [String!]
+  "Call data for the change"
+  calldatas: [Bytes!]
+}
+
+enum ProposalState {
+  PENDING
+  ACTIVE
+  CANCELED
+  DEFEATED
+  SUCCEEDED
+  QUEUED
+  EXPIRED
+  EXECUTED
+}
+
+type Vote @entity {
+  "Delegate ID + Proposal ID"
+  id: ID!
+  "Whether the vote is in favour, against or abstaining to the proposal"
+  choice: VoteChoice!
+  "Voting weight expressed in the vote"
+  weight: BigInt!
+  "Reason for voting choice"
+  reason: String
+  "Delegate that emitted the vote"
+  voter: Delegate!
+  "Proposal that is being voted on"
+  proposal: Proposal!
+}
+
+enum VoteChoice {
+  FOR
+  AGAINST
+  ABSTAIN
+}
+
+type TokenHolder @entity {
+  "A TokenHolder is any address that holds any amount of Compound Tokens, the id used is the blockchain address."
+  id: ID!
```

Comment:
Consider using `Bytes` for performance reasons: https://medium.com/edge-node-engineering/two-simple-subgraph-performance-improvements-a76c6b3e7eac